src/applications/vaos/actions/newAppointment.js

```diff
@@ -389,7 +390,13 @@ export function openSelectAppointmentPage(page, uiSchema, schema) {
 
     try {
       const response = await getAvailableSlots(
-        getState().newAppointment.data.clinicId,
+        data.vaFacility,
+        data.typeOfCareId,
+        data.clinicId,
+        moment().format('YYYY-MM-DD'),
```

Comment:
Are these dates right? Or should we start with their requested date?

Also, I think we'll have to change how this works to fetch slots 90+ days out as the user gets to those months.

Reply:
This needs to be updated.  Probably with start date being the first of the month of their preferred date's month, and end date being the end of the following month.  

I haven't updated this to fetch in chunks yet, but we should probably do a fetch when the user hits next/prev, if data for those months haven't been fetched yet.  I can address it in a new ticket since I'm more familiar.  Up to you.

Reply:
Fetching in chunks should definitely be a separate ticket, but I can update the start date here.

Reply:
@narin I updated both dates to take the preferred date into account, and created a ticket for fetching further slots.